src/main/webapp/assets/css/feedStyle.css

```diff
@@ -240,6 +246,22 @@ select {
   outline: none;
 }
 
+.modal-date-tip-text {
+  color: #07536a;
```

Comment:
Maybe var(--light-teal) or var(--dark-green) to stay in our color palette?

Reply:
Unfortunately, both of those colors are too light to be accessible at the given font size:
![image](https://user-images.githubusercontent.com/35010111/91457048-cd654480-e838-11ea-9bd3-20ad38c4e924.png)
![image](https://user-images.githubusercontent.com/35010111/91457243-fc7bb600-e838-11ea-9d2c-4ac05cf5025b.png)
![image](https://user-images.githubusercontent.com/35010111/91457187-ef5ec700-e838-11ea-89b3-bbaff92aadd9.png)

The only color in our color palette that has sufficient contrast is var(--dark-grey), which felt a little boring to me (it doesn't really draw the eye as much), so I ended up going with a dark teal as a compromise between var(--light-teal) and var(--dark-green).
![image](https://user-images.githubusercontent.com/35010111/91457528-4c5a7d00-e839-11ea-9c16-532f9a4f4b78.png)

Reply:
I think dark green should be fine since the submit button is also dark green!

Reply:
@ubahl, @areetaw, I can switch it to green, however it'll have to be made darker to be accessible :) Please let me know whether you just want me to make it strictly darker (`2`) or darker and more vibrant to compensate (`3`)!

1. var(--dark-green), which is not accessible at that font size (AAA)
![image](https://user-images.githubusercontent.com/35010111/91463545-5fbd1680-e840-11ea-9d47-3d9c599bc53b.png)
2. var(--dark-green), made darker so that it becomes accessible
![image](https://user-images.githubusercontent.com/35010111/91463616-75324080-e840-11ea-986c-af5963ca4f1d.png)
3. var(--dark-green), made darker so that it becomes accessible but also more vibrant so that it's clear that it is green
![image](https://user-images.githubusercontent.com/35010111/91463754-9d21a400-e840-11ea-8bee-22ac9fc2d5c6.png)

Reply:
#3 looks good!